forage-android/src/main/java/com/joinforage/forage/android/ui/InternalForageElement.kt

```diff
@@ -0,0 +1,26 @@
+package com.joinforage.forage.android.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.LinearLayout
+
+abstract class InternalForageElement(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : LinearLayout(context, attrs, defStyleAttr), ForageElement {
+
+    private var _forageContext: ForageContext? = null
+    override fun setForageContext(forageContext: ForageContext) {
+        this._forageContext = forageContext
+    }
+
+    // internal because submit methods need read-access
+    // to the ForageContext but not public because we
+    // don't to lull developers into passing a
+    // ForagePANEdText instance around as a proxy for
```

Comment:
typo

Reply:
🤦

Reply:
fixed in [5dcb8b3](https://github.com/teamforage/forage-android-sdk/pull/97/commits/5dcb8b36a2906318b3650c39eb8fc37a85139a3b). Love the attention to detail haha